R/pin_manifest.R

```diff
@@ -0,0 +1,66 @@
+#' Write manifest file
+#'
+#' This can be used if you have a `pins::board_folder()` that you want to
```

Comment:
Probably should not specifically call out `board_folder()` here since it will be extended

Reply:
agreed!